test/service/test_drs.py

```diff
@@ -0,0 +1,99 @@
+import requests
+import unittest
+import sys
+
+from service import WebServiceTestCase
+
+sys.path.insert(0, ".")
+
+from lambdas.service.app import azul_to_obj
+
+
+class DataRepositoryServiceEndpointTest(WebServiceTestCase):
+
+    def get_data_object(self):
+        """
+        Helper function to get a data object using the
+        repository/files list feature.
+        :return:
+        """
+        url = self.base_url + "/repository/files"
+        response = requests.get(url)
+        response.raise_for_status()
+        response_json = response.json()
+        file_id = response_json['hits'][0]['files'][0]['uuid']
+        get_url = "{}/ga4gh/dos/v1/dataobjects/{}".format(self.base_url, file_id)
+        drs_response = requests.get(get_url)
+        drs_response.raise_for_status()
+        drs_response_json = drs_response.json()
+        data_object = drs_response_json['data_object']
+        return data_object, file_id
+
+    def test_get_data_object(self):
+        """
+        Ensures that a file requested via repository can also
+        be found via DRS and that the returned document shares
+        the same identifier.
+        :return:
+        """
+        data_object, file_id = self.get_data_object()
+        self.assertEqual(file_id, data_object['id'], "The IDs should match")
+
+    def test_data_object_not_found(self):
+        """
+        Ensures that when an unused identifier is requested a 404 status
+        code is returned.
+        :return:
+        """
+        file_id = "NOT_A_GOOD_IDEA"
+        get_url = "{}/ga4gh/dos/v1/dataobjects/{}".format(self.base_url, file_id)
+        # Should cause a 404 error
+        with self.assertRaises(requests.exceptions.HTTPError):
+            drs_response = requests.get(get_url)
+            drs_response.raise_for_status()
+
+    @unittest.skip("DSS and Azul indices do not match in testing"
```

Comment:
Reading this, I get the feeling that the tests are actually hitting the DSS. That would disqualify them as unit tests, obviously. If I am correct and if at all possible, try mocking the DSS responses. test_dss_proxy.py does that.